tox.ini

```diff
@@ -110,12 +110,30 @@ commands =
 [testenv:py27-numpy]
 deps=numpy
 commands=
-  pytest -rfsxX {posargs:testing/python/approx.py}
+  pytest -ra {posargs:testing/python/approx.py}
 
 [testenv:py35-numpy]
 deps=numpy
 commands=
-  pytest -rfsxX {posargs:testing/python/approx.py}
+  pytest -ra {posargs:testing/python/approx.py}
+
+[testenv:py27-pluggymaster]
+passenv={[testenv]passenv}
+commands={[testenv]commands}
+setenv=
+    _PYTEST_SETUP_SKIP_PLUGGY_DEP=1
+deps =
+    {[testenv]deps}
+    git+https://github.com/pytest-dev/pluggy.git@master
+
+[testenv:py35-pluggymaster]
+passenv={[testenv:py27-pluggymaster]passenv}
+commands={[testenv:py27-pluggymaster]commands}
+setenv=
+    _PYTEST_SETUP_SKIP_PLUGGY_DEP=1
+deps =
```

Comment:
@nicoddemus heh I was actually thinking to derive `setenv` and `deps` just because they're common to these two envs. 

Not a big deal though :)